src/tasks/AutoCinemaTask.py

```
@@ -0,0 +1,175 @@
+            self.send_key('d', down_time=0.61, after_sleep=3.78)
+        finally:
+            self.send_key_up('w', after_sleep=2.16)
+
+    # 选择邀约电影对象
+    def select_date(self):
+        have_date = False
+        regex_str = self.config.get(self.CONF_MOVIE_CHAR,"薄荷")
+        try:
+            match_regex = re.compile(regex_str)
+        except re.error as e:
+            raise RuntimeError(f"邀约角色配置无效: {regex_str} ({e})")
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)